### PR TITLE
fix PathServiceTest.Get test

### PR DESCRIPTION
**Pull request by victorpasoshnikov**
Test was modified according to XBOX limitations

b/219026606

Change-Id: Iea811495b50fd29588ba439a955d9522672adb88

### CODE REVIEW

**Comment by victorpasoshnikov** [comment]
CL 226181

File: base/path_service_unittest.cc [hunk 1]
```diff
@@ -57,7 +57,10 @@ bool ReturnsValidPath(int dir_type) {
   if (path.ReferencesParent())
     return false;
 #endif
-  return result && !path.empty() && (!check_path_exists || PathExists(path));
+  // Some existing root folders (LocalFolder, LocalCacheFolder, etc.)
```
Comment (kaidokert) [inline base/path_service_unittest.cc:60]: When reading the file above, it looks like the general pattern is 
```
#if PORT 
  if (dir_type == DIR_ASSETS)
    check_path_exists = false;
#endif
```
If you add this section above
```
#if defined(STARBOARD)
  if (dir_type == DIR_XYZ) // i don't know what goes here exactly
    check_path_exists = false;
#endif // defined(STARBOARD)
```
Does it fix the issue similarly ? If yes, this approach would be preferable as it doesn't modify common, platform specific upstream Chrome code.

**Review by kaidokert — CHANGES_REQUESTED** [review]

**Comment by kaidokert** [comment]
@TyHolc ping for a review / comments / thoughts ?

**Comment by TyHolc** [comment]
> @TyHolc ping for a review / comments / thoughts ?

If we're sure this will be non-disruptive to other platforms by expanding the scope of "does this path exist" to include directories, then I'm fine with this. 

An alternative could be:

```
#if defined(OS_WIN)
  if (dir_type == <LocalFolder> || dir_type == <LocalCacheFolder> || dir_type == <etc>...) {
    if (!DirectoryExists(path)) {
      return false;
    }
    check_path_exists = false;
  }
#endif
```

**Comment by TyHolc** [comment]
I'm going to leave this test disabled on Xbox for now and close this. Skipping this test with test filters for xbox is preferable to me over having all platforms skip checking their implementation of `Get` for their respective home directories.

**Pull request closed without merging**